ddtrace/llmobs/_experiment.py

```diff
@@ -71,6 +73,23 @@ def __init__(
         self._id: Optional[str] = None
 
     def run(self, jobs: int = 1, raise_errors: bool = False, sample_size: Optional[int] = None) -> None:
+        if not self._llmobs_instance or not self._llmobs_instance.enabled:
+            raise ValueError(
+                "LLMObs is not enabled. Ensure LLM Observability is enabled via `LLMObs.enable(...)` "
+                "and create the experiment via `LLMObs.experiment(...)` before running the experiment."
+            )
+        experiment_id, experiment_name = self._llmobs_instance._create_experiment(
+            name=self.name,
+            dataset_id=self._dataset._id,
+            project_name=self._project_name,
+            dataset_version=self._dataset._version,
+            exp_config=self._config,
+            tags=self._tags,
+            description=self._description,
+        )
+        self._id = experiment_id
+        if experiment_name != self.name:
+            self.name = experiment_name
```

Comment:
should we log a warning about dupe experiment names in this case?

Reply:
Yeah good callout. Will add a debug log

Reply:
I'm not seeing run-name or anything similar (name + timestamp) in the experiment create response. I might make this a follow up

Reply:
nvm - realized run_name is returned as name, and experiment name is returned as experiment. Made the changes here after all